### PR TITLE
kubernetes-nmstate:added node config examples to README.md

### PR DESCRIPTION
**Pull request by vakhanna**
# Description

feature kubernetes-nmstate: added node configuration examples to README.m

Fixes # (issue)

## Type of change

Please select the appropiate options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Testing

Since its documentation enhancement, no testing is required

## Checklist

- [x] I have performed a self-review of my own code
- [x] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not being merged unless it passes CI or there is a comment/update saying what testing was passed.
- [ ] PRs should not be merged unless positively reviewed.

### CODE REVIEW

**Comment by vakhanna** [comment]
@SchSeba please review.

File: features/kubernetes-nmstate/README.md [hunk 1]
```diff
@@ -80,4 +80,46 @@ To deploy kubernetes-nmstate use the `./deploy.sh` script in this folder.
...
 The test will create a bond interface with the name `bond1` and attach `ens8` and `ens9` as the slave interfaces.
 
-*Note:* You can change the bond name and the slave interface names by editing the `myenv.file` file.
\ No newline at end of file
+*Note:* You can change the bond name and the slave interface names by editing the `myenv.file` file.
+
+### Node configuration examples
+
+```
+# IP configuration
```
Comment (e-minguez) [inline features/kubernetes-nmstate/README.md]: I'd say to move this out of the triple quotes and convert it to a list:

```
* IP configuration
```

File: features/kubernetes-nmstate/README.md [hunk 2]
```diff
@@ -80,4 +80,46 @@ To deploy kubernetes-nmstate use the `./deploy.sh` script in this folder.
...
+
+10: bond1: <BROADCAST,MULTICAST,MASTER,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
+    link/ether 1c:40:24:99:5d:0d brd ff:ff:ff:ff:ff:ff
```
Comment (e-minguez) [inline features/kubernetes-nmstate/README.md]: IDK if you want to remove the MAC addresses here

File: features/kubernetes-nmstate/README.md [hunk 3]
```diff
@@ -80,4 +80,46 @@ To deploy kubernetes-nmstate use the `./deploy.sh` script in this folder.
...
+    inet 192.168.10.100/24 brd 192.168.10.255 scope global noprefixroute bond1
+       valid_lft forever preferred_lft forever
+11: bond1.20@bond1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
+    link/ether 1c:40:24:99:5d:0d brd ff:ff:ff:ff:ff:ff
+    inet 192.168.20.100/24 brd 192.168.20.255 scope global noprefixroute bond1.20
+       valid_lft forever preferred_lft forever
+
+# Bond configuration
```
Comment (e-minguez) [inline features/kubernetes-nmstate/README.md]: Same comment than the previous IP configuration thing

File: features/kubernetes-nmstate/README.md [hunk 4]
```diff
@@ -80,4 +80,46 @@ To deploy kubernetes-nmstate use the `./deploy.sh` script in this folder.
...
+MII Polling Interval (ms): 140
+Up Delay (ms): 0
+Down Delay (ms):
+
+Slave Interface: enp3s0f2
+MII Status: up
+Speed: 1000 Mbps
+Duplex: full
+Link Failure Count: 0
+Permanent HW addr: 1c:40:24:99:5d:0d
```
Comment (e-minguez) [inline features/kubernetes-nmstate/README.md]: Same here for those MAC addresses

File: features/kubernetes-nmstate/README.md [hunk 5]
```diff
@@ -80,4 +80,46 @@ To deploy kubernetes-nmstate use the `./deploy.sh` script in this folder.
...
+Slave queue ID: 0
+
+Slave Interface: enp3s0f3
+MII Status: up
+Speed: 1000 Mbps
+Duplex: full
+Link Failure Count: 0
+Permanent HW addr: 1c:40:24:99:5d:0e
+Slave queue ID: 0
+
```
Comment (e-minguez) [inline features/kubernetes-nmstate/README.md]: Unneeded new line

**Comment by SchSeba** [comment]
/cc @phoracek @qinqon

Can you take a look on this?

File: features/kubernetes-nmstate/README.md [hunk 6]
```diff
@@ -80,4 +80,46 @@ To deploy kubernetes-nmstate use the `./deploy.sh` script in this folder.
...
+*Note:* You can change the bond name and the slave interface names by editing the `myenv.file` file.
+
+### Node configuration examples
+
+```
+# IP configuration
+
+10: bond1: <BROADCAST,MULTICAST,MASTER,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
+    link/ether 1c:40:24:99:5d:0d brd ff:ff:ff:ff:ff:ff
+    inet 192.168.10.100/24 brd 192.168.10.255 scope global noprefixroute bond1
```
Comment (qinqon) [inline features/kubernetes-nmstate/README.md]: Let's use doc ip address ranges from https://tools.ietf.org/html/rfc5737

File: features/kubernetes-nmstate/README.md (hunk 1 above)
Comment (vakhanna) [reply features/kubernetes-nmstate/README.md]: thanks, done.

File: features/kubernetes-nmstate/README.md (hunk 2 above)
Comment (vakhanna) [reply features/kubernetes-nmstate/README.md]: masked both macs and IPs.
fyi, there are mac and IP addresses listed in NodeNetworkState section also, please open an issue so the right person can address that too.

File: features/kubernetes-nmstate/README.md (hunk 5 above)
Comment (vakhanna) [reply features/kubernetes-nmstate/README.md]: This is to be expected, reflecting command output here.

File: features/kubernetes-nmstate/README.md (hunk 6 above)
Comment (vakhanna) [reply features/kubernetes-nmstate/README.md]: I've masked the IPs to avoid any inconsistency. thanks.

**Comment by vakhanna** [comment]
@e-minguez @qinqon Thanks for feedback, please review the changes.

**Comment by e-minguez** [comment]
I think you need to rebase your PR as it has a lot of other unrelated commits.

**Pull request merged**